### PR TITLE
fixed windows platform error

### PR DESCRIPTION
**Pull request by mcroni**

### CODE REVIEW

File: kivystudio/widgets/codeinput/__init__.py [hunk 1]
```diff
@@ -47,10 +47,7 @@ class InnerCodeInput(HoverBehavior, CodeInput):
     def __init__(self, **kwargs):
         super(InnerCodeInput, self).__init__(**kwargs)
         self.rightclick_dropdown = CodeInputDropDown(self)
-        try:
-            self.style_name = 'native_tweak'
-        except ValueError:
-            self.style_name = 'colorful'
```
Comment (avour) [inline kivystudio/widgets/codeinput/__init__.py]: sorry for this native tweak problems it was a tweak i was working on and didn't push so it caused problems

**Comment by avour** [comment]
Thanks for contributing bro!

File: kivystudio/widgets/codeinput/__init__.py (hunk 1 above)
Comment (mcroni) [reply kivystudio/widgets/codeinput/__init__.py]: no problem Oga

Comment (mcroni) [reply kivystudio/widgets/codeinput/__init__.py]: lets make a todo on the important stuffs to implement so we dont work on the same stuffs at the same time

**Comment by avour** [comment]
Am working on one now but for now you could a custom popup and notification
for the studio.
and by the way nice video you got on YouTube

**Comment by avour** [comment]
You could just pick one from the list

**Pull request merged**